nodejs-server/routes/photos.js

```diff
@@ -0,0 +1,78 @@
+var counter  = require('counter');
+var express = require('express');
+var fs = require('fs');
+var multer  = require('multer');
+var path = require('path');
+var router = express.Router();
+
+var MQTT = require('../src/MQTT');
+var DI = require('../src/DI');
```

Comment:
is this used to get photo directory?

Reply:
I wasn't sure where to store such things in the js folder structure so I make a DI (dependency￼ injection) file for instantiating classes with constants like the file directory of where the photos are saved